packages/devtools_app/test/flutter/timeline_model_test.dart

```diff
@@ -0,0 +1,317 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
welcome to 2020 here and elsewhere.
Please also mark tests that should be delete when the dart:html app is removed as part of this CL.

Reply:
I left it as 2019 because I copied it from test/timeline_model_test.dart.